Deployment.md

```diff
@@ -65,7 +65,7 @@ The following resources are deployed by [deployCluster.sh](deployment/deployClus
 - Azure Traffic Manager endpoint to cluster
 
 The following diagram shows the **physical artifacts** created in the resource group.
-![](./deployment/kubernetes-resource-group.png)
+![Kubernetes Resources](./deployment/kubernetes-resource-group.png)
```

Comment:
"etcd" mispelled in image
![image](https://user-images.githubusercontent.com/2847527/39491957-f65dc512-4d5b-11e8-9394-c7d09aff55d5.png)

Reply:
Also, I would change the "docker" layer to "kubelet".

Reply:
We could mimic this diagram for the Kube setup bits. I'd leave off cAdvisor

https://upload.wikimedia.org/wikipedia/commons/b/be/Kubernetes.png

Reply:
@jobanputra created the diagram above in a separate a while back. @billpratt could you open a bug in VSTS for those items?

Reply:
Will do

Reply:
I created another pull request with changes.